.golangci.yml

```diff
@@ -21,11 +22,15 @@ linters:
   - goimports
   - gomodguard
   - gosec
+  - govet
   - gocritic
   - revive
   - misspell
   - unconvert
   - depguard
+    # To keep this PR small, I'm going to add one linter at a time.
+    #- unused
+    #- whitespace
```

Comment:
discussed offline - these are TODOs

Reply:
Yes, I plan to follow this PR with these changes; trying to keep the PRs small and digestible since there are related (non-functional) code changes.

Reply:
Thank you. Do you want to create an issue to track these unless you plan to tackle them right away